docs/Common/Structs/index.md

```diff
@@ -818,3 +818,184 @@
+|servicePublished|Boolean|true||If true, the service is published and available. If false, the service has likely just been unpublished, and should be considered unavailable|
+|serviceActive|Boolean|true||If true, the service is the active primary service of the supplied service type. It will receive all potential RPCs that are passed through to that service type. If false, it is not the primary service of the supplied type. See servicePublished for its availability|
+
+### AppServiceData
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|serviceType|String|true||The type of service that is to be offered by this app. See AppServiceType for known enum equivalent types. Parameter is a string to allow for new service types to be used by apps on older versions of SDL Core|
+|serviceID|String|true||A unique ID tied to this specific service|
+|mediaServiceData|Common.MediaServiceData|false|||
+|weatherServiceData|Common.WeatherServiceData|false|||
+|navigationServiceData|Common.NavigationServiceData|false|||
+
+### AppServiceCapability
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|updateReason|[Common.ServiceUpdateReason](../enums/#serviceupdatereason)|false||Only included in OnSystemCapabilityUpdated. Update reason for service record|
+|updatedAppServiceRecord|Common.AppServiceRecord|true||Service record for a specific app service provider|
+
+### AppServicesCapabilities
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|appServices|Common.AppServiceCapability|false|array: true|An array of currently available services. If this is an update to the capability the affected services will include an update reason in that item|
+
+### SystemCapability
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|systemCapabilityType|[Common.SystemCapabilityType](../enums/#systemcapabilitytype)|true||Used as a descriptor of what data to expect in this struct. The corresponding param to this enum should be included and the only other param included|
+|navigationCapability|Common.NavigationCapability|false||Describes extended capabilities for onboard navigation system|
+|phoneCapability|Common.PhoneCapability|false||Describes extended capabilities of the module's phone feature|
+|videoStreamingCapability|Common.VideoStreamingCapability|false||Describes extended capabilities of the module's phone feature|
+|remoteControlCapability|Common.RemoteControlCapabilities|false||Describes extended capabilities of the module's phone feature|
+|appServicesCapabilities|Common.AppServicesCapabilities|false||An array of currently available services. If this is an update to the capability the affected services will include an update reason in that item|
+
+### MediaServiceManifest
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+||||||
```

Comment:
Maybe remove the table and leave a note saying `There are no defined parameters for this struct.`